src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2752,25 +2752,25 @@ async void M()
         [Fact]
         public void TestAwaitUsingVarWithVarAndNoUsingDeclarationTree()
         {
-            UsingStatement(@"await var a = b;", TestOptions.Regular8, expectedErrors:
-                // (1,11): error CS1003: Syntax error, ',' expected
+            UsingStatement(@"await var a = b;", TestOptions.Regular8,
+                // (1,1): error CS1073: Unexpected token 'a'
```

Comment:
hard to even tell what we would want here.  We could potentially improve parsing await exressions to only consume the `await` and not consume into the local decl.  

Note that hte previous parsing was also very bad.  it parsed this as `await var`.  i.e. a local called `var` of type `await`.  The rest of the expression then was just skipped.  So neither approach is really a win here.